### DISCUSSION TITLE
对于启用5秒护盾的网站，人机认证时一直重定向，无法访问网站主界面

### DISCUSSION THREAD

**Discussion by abbtiktok** [SeaHOH/GotoX#195] [一般]
例如这个网站：https://temp-mail.org/ ，请问有解决办法吗？

**Comment by SeaHOH** [comment]
本工具只是个代理而已，和这种情况有什么关系吗？
据我的经验，起码要进行两遍，这套系统好像就是这么无赖。如果确实始终无法通过验证，建议换一个干净的浏览器配置试试，可能是某个扩展或插件的相关兼容性问题。
另外，注意看页面提示，验证方还有提供浏览器扩展帮助跳过手动验证。
